eth/downloader/statesync.go

```diff
@@ -59,6 +59,13 @@ type stateSyncStats struct {
 
 // syncState starts downloading state with the given root hash.
 func (d *Downloader) syncState(root common.Hash) *stateSync {
+	// The downloader was requested to start a state sync. If the state sync bloom
+	// filter was not yet initialized, create it now. This lazy creation ensures we
+	// only allocate if if really really really needed.
```

Comment:
```suggestion
	// only allocate if really really really needed.
```
`3` x `really` looks intentional, but `2` x `if` looks like a bug